src/app/shared/pipes/groupLink.ts

```diff
@@ -4,12 +4,12 @@ import { RawGroupRoute } from '../routing/group-route';
 import { urlArrayForGroupRoute } from '../routing/group-route';
 
 /**
- * Functions using item route should always be preferred to raw item.
- * Using raw item means further requests will be required to fetch path and attempt information.
+ * Pipe to format group links, with or without the path
+ * Since a user is considered as a group of one person, user routes are formatted using this pipe too.
  */
 @Pipe({ name: 'groupLink', pure: true })
 export class GroupLinkPipe implements PipeTransform {
   transform(route: RawGroupRoute, isUser?: boolean): UrlCommand {
```

Comment:
BTW, why a `isUser?` which supports then 3 values: `true`, `false`, `undefined` while I suppose there is only the semantic for 2 values. If it is false when not set, just use a default value for the parameter.

Reply:
Fixed